x-pack/plugins/spaces/server/lib/request_interceptors/on_post_auth_interceptor.test.ts

```diff
@@ -8,10 +8,11 @@
 import Boom from '@hapi/boom';
 
 // @ts-ignore
+import type { createRoot } from '@kbn/core-test-helpers-kbn-server';
```

Comment:
@elastic/kibana-security  I split the imports out into separate methods and/or types after migrating the `kbn_server` utilities to a package.

Reply:
You could technically also have done this to the same effect:

```ts
import {
  type createRoot,
  request as kbnTestServerRequest
} from '@kbn/core-test-helpers-kbn-server';
```

Reply:
True. Done